relevanceai/http_client.py

```diff
@@ -100,3 +101,32 @@ def check_auth(self):
             return response.status_code == 200    
         except:
             raise Exception("Invalid auth details.")
+
+    def kmeans_cluster(self, dataset_id, vector_fields, k = 10, update_dataset = True):
```

Comment:
Missing: 
- Filters for retrieving documents 
- Arguments for clustering on KMeans

Reply:
- Alias as well for the clustering and storing of centroids

Reply:
Done!